src/responders/seat.ts

```diff
@@ -0,0 +1,41 @@
+import { Message, MessageEmbed } from 'discord.js'
+import * as log from '../lib/console'
+import axios from 'axios'
+
+const seatdocker = getVersion(`https://img.shields.io/docker/v/eveseat/seat.json`)
+const api = getVersion(`https://img.shields.io/github/v/release/eveseat/api.json`)
+const seat_console = getVersion(`https://img.shields.io/github/v/release/eveseat/console.json`)
+const eveapi = getVersion(`https://img.shields.io/github/v/release/eveseat/eveapi.json`)
+const notifications = getVersion(`https://img.shields.io/github/v/release/eveseat/notifications.json`)
+const services = getVersion(`https://img.shields.io/github/v/release/eveseat/services.json`)
+const web = getVersion(`https://img.shields.io/github/v/release/eveseat/web.json`)
```

Comment:
Does it not make sense to rather use the Github API here instead of this dependency? For example <https://api.github.com/repos/eveseat/web/releases/latest> returns the `tag_name` property for this:

```json
{
  "url": "https://api.github.com/repos/eveseat/web/releases/56378880",
  "assets_url": "https://api.github.com/repos/eveseat/web/releases/56378880/assets",
  "upload_url": "https://uploads.github.com/repos/eveseat/web/releases/56378880/assets{?name,label}",
  "html_url": "https://github.com/eveseat/web/releases/tag/4.17.0",
  "id": 56378880,
  "author": {
    "login": "warlof",
    "id": 648753,
    "node_id": "MDQ6VXNlcjY0ODc1Mw==",
    "avatar_url": "https://avatars.githubusercontent.com/u/648753?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/warlof",
    "html_url": "https://github.com/warlof",
    "followers_url": "https://api.github.com/users/warlof/followers",
    "following_url": "https://api.github.com/users/warlof/following{/other_user}",
    "gists_url": "https://api.github.com/users/warlof/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/warlof/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/warlof/subscriptions",
    "organizations_url": "https://api.github.com/users/warlof/orgs",
    "repos_url": "https://api.github.com/users/warlof/repos",
    "events_url": "https://api.github.com/users/warlof/events{/privacy}",
    "received_events_url": "https://api.github.com/users/warlof/received_events",
    "type": "User",
    "site_admin": false
  },
  "node_id": "RE_kwDOApGD084DXEYA",
  "tag_name": "4.17.0",
  "target_commitish": "master",
  "name": "v4.17.0",
  "draft": false,
  "prerelease": false,
  "created_at": "2022-01-03T16:12:41Z",
  "published_at": "2022-01-04T13:55:18Z",
  "assets": [

  ],
  "tarball_url": "https://api.github.com/repos/eveseat/web/tarball/4.17.0",
  "zipball_url": "https://api.github.com/repos/eveseat/web/zipball/4.17.0",
  "body": "## What's Changed\r\n* fix: update moon chunk quantity by @Crypta-Eve in https://github.com/eveseat/web/pull/589\r\n* fix: address a typo in request injection by @warlof in https://github.com/eveseat/web/pull/590\r\n* fix: show coin icon on dashboard by @warlof in https://github.com/eveseat/web/pull/591\r\n\r\n\r\n**Full Changelog**: https://github.com/eveseat/web/compare/4.16.0...4.17.0",
  "mentions_count": 2
}
```